PyInstaller/building/build_main.py

```diff
@@ -119,6 +119,39 @@ def discover_hook_directories():
     return hook_directories
 
 
+# NOTE: this helper is called via isolated.call() in Analysis.assemble(). We cannot use @isolated.decorate here, because
+# one of the tests (test_regression::test_issue_5131) needs to be able to monkey-patch it, in order to override the
+# bindepend.getImports() in the isolated subprocess (!) with its own implementation...
+def find_binary_dependencies(binaries, binding_redirects, import_packages):
+    """
+    Find dynamic dependencies (linked shared libraries) for the provided list of binaries.
+
+    Before scanning the binaries, the function imports the packages from provided list of packages to import, to ensure
+    that library search paths are properly set up (i.e., if a package sets up search paths when imported9. Therefore,
```

Comment:
I'm guessing you meant:
```suggestion
    that library search paths are properly set up (i.e., if a package sets up search paths when imported). Therefore,
```

Reply:
Yeah. Fixed.